src/config/ConfigManager.cpp

```diff
@@ -34,6 +34,8 @@ void CConfigManager::setDefaultVars() {
     configValues["general:apply_sens_to_raw"].intValue = 0;
     configValues["general:main_mod"].strValue = "SUPER";                                               // exposed to the user for easier configuring
     configValues["general:main_mod_internal"].intValue = g_pKeybindManager->stringToModMask("SUPER");  // actually used and automatically calculated
+    configValues["general:workspace_back_and_forth"].intValue = 0;
```

Comment:
if

> Also included is a toggle setting general:workspace_back_and_forth to allow switching back to the previous workspace by trying to switch to the same workspace again. I use it all the time in sway and my workflow would be greatly improved by the ability to quickly switch to and from a workspace to check on email, discord, spotify, etc.

wouldn't it be appropriate to put this in `binds:`?

Reply:
Good idea.